.github/workflows/chromatic.yml

```
@@ -8,13 +8,14 @@ jobs:
     if: "!contains(github.event.head_commit.message, 'chore(release): publish')"
     strategy:
       matrix:
-        node-version: [10.16.x]
+        node-version: [12.18.x]
 
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: actions/setup-node@v1
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/setup-node@v2-beta
```

I think you can remove the line 17 `node-version`, and put is under the `setup-node` action like so. Saves having to use the beta of v2 of `setup-node`

```suggestion
      - uses: actions/setup-node@v1
        with:
          node-version: ${{ matrix.node-version }}
```

---

I think I resolved it now, let me know